package/ci/appveyor-desktop-mingw.bat

```diff
@@ -60,13 +60,16 @@ cd .. && cd ..
 
 rem Build. BUILD_GL_TESTS is enabled just to be sure, it should not be needed
 rem by any plugin.
+rem WITH_BASISIMPORTER is disabled, since Basis Universal currently does not
+rem support building on MinGW.
```

Comment:
I just realized why https://ci.appveyor.com/project/mosra/magnum-plugins/builds/26945769/job/wk4pvpxw8d5s1y03#L1117 didn't build but https://ci.appveyor.com/project/mosra/magnum-plugins/builds/27017597/job/fdb7jx0qxspnox4p did -- originally you had the whole thing embedded and now you're building just the transcoder part. So that means the `Importer` could be enabled on MinGW, but the `ImageConverter` in #65 can't be. Enable this again then and move this comment to #65 then? :)

Reply:
I think I had enabled this in the third or second to last build though to make sure ... And it didn't build...

Reply:
But for a different reason ;) https://ci.appveyor.com/project/mosra/magnum-plugins/builds/27048449/job/ayx4ravm2almu2s8 ... because the YML doesn't download the library for MinGW.

Reply:
Oh! I thought I disabled that for that build 🙈 good catch, should be done around 4pm today at the latest